apps.md

```diff
@@ -77,4 +77,21 @@ TimeTrapp supports:
 
 * * *
 
+## Kimai-widget - Rainmeter skin <i class="fa fa-windows"></i>
+
+Rainmeter skin for Kimai, you can start and stop the timer for a selected project and activity. Requires Rainmeter, a desktop customization tool for Windows, download and more info: [rainmeter.net](https://www.rainmeter.net/)
```

Comment:
Can you please remove the link to Rainmeter here and instead add all necessary information to your repositories README? It might happen that users find your repository without this page and need the information as well.